src/components/Navigation/Navigation.tsx

```diff
@@ -0,0 +1,12 @@
+import { useTranslation } from 'react-i18next';
+import { Outlet } from 'react-router-dom';
+
+export const Navigation = (): JSX.Element => {
+  const { t } = useTranslation();
+  return (
```

Comment:
Placeholder na nawigację, pytanie jakie mamy podejście do komponentów - z podziałem na index itp czy jako jeden plik `Navigation.tsx` z kompoentem ?

Reply:
I think we can use index.ts convention in case there are multiple files as parts of 1 component. If there is only 1 file then I think there is no need for index.ts (it is easily changeable to index.ts pattern in case this component needs to be divided)